suit_rq/templates/django_rq/job_detail.html

```diff
@@ -0,0 +1,110 @@
+{% extends "admin/base_site.html" %}
+
+{% load url from future %}
+
+{% block breadcrumbs %}
+    <ul class="breadcrumb">
+        <li>
+            <a href="{% url 'admin:index' %}">Home</a>
+            <span class="divider">»</span>
+        </li>
+        <li>
+            <a href="{% url 'rq_home' %}">Django RQ</a>
+            <span class="divider">»</span>
+        </li>
+        <li>
+            <a href="{% url 'rq_jobs' queue_index %}">{{ queue.name }}</a>
+            <span class="divider">»</span>
+        </li>
+        <li class="active">
+            <a href="{% url 'rq_job_detail' queue_index job.id %}">{{ job.id }}</a>
+        </li>
+    </ul>
+{% endblock %}
+
+{% block content_title %}<h1>Job Info</h1>{% endblock %}
+
+{% block content %}
+
+<div id="content-main">
+
+    <div class="module">
+
+        <table class="table table-striped table-bordered table-hover table-condensed">
+            <tbody>
+                <tr>
+                    <th>ID:</th>
+                    <td>{{ job.id }}</td>
+                </tr>
+                <tr>
+                    <th>Queue:</th>
+                    <td>{{ job.origin }}</td>
+                </tr>
+                <tr>
+                    <th>Timeout:</th>
+                    <td>{{ job.timeout }}</td>
+                </tr>
+                <tr>
+                    <th>Result TTL:</th>
+                    <td>{{ job.result_ttl }}</td>
+                </tr>
+                <tr>
+                    <th>Created:</th>
+                    <td>{{ job.created_at }}</td>
+                </tr>
+                <tr>
+                    <th>Queued:</th>
+                    <td>{{ job.enqueued_at }}</td>
+                </tr>
+                <tr>
+                    <th>Status:</th>
+                    <td>{{ job.get_status }}</td>
+                </tr>
+                <tr>
+                    <th>Callable:</th>
+                    <td>{{ job.func_name }}</td>
+                </tr>
+                <tr>
+                    <th>Args:</th>
+                    <td>
+                    {% if job.args %}
+                    <ul>
+                        {% for arg in job.args %}
+                            <li>{{ arg }}</li>
+                        {% endfor %}
+                    </ul>
+                    {% endif %}
+                </td>
+                </tr>
+                <tr>
+                    <th>Kwargs:</th>
+                    <td>
+                    {% if job.kwargs %}
+                        <ul>
+                            {% for key, value in job.kwargs.items %}
+                                <li>{{ key }}: {{ value|escape }}</li>
+                            {% endfor %}
+                        </ul>
+                    {% endif %}</td>
+                </tr>
+                <tr>
```

Comment:
Can you move the `{% if job.exc_info %}` to be around the whole ``<tr>`` here to prevent it from even showing up if there wasn't an error?